### ISSUE TITLE
Locked users list filter

### ISSUE THREAD

**Issue by soullivaneuh** [FusionAuth/fusionauth-issues#688]
### Problem

In order to do some analysis, I would like to list quickly all the **locked** account.

### Solution

A a flag filter to display All/Only locked/Only unlocked filter here:

![image](https://user-images.githubusercontent.com/1698357/84882405-80722e80-b08f-11ea-855c-c1319cdc9004.png)

### Alternatives/workarounds

Not so much instead doing some database queries.

### How to vote
Please give us a thumbs up or thumbs down as a reaction to help us prioritize this feature. Feel free to comment if you have a particular need or comment on how this feature should work.

**Comment by robotdan** [comment]
When using Elasticsearch, you can search using `active:false`. 

If you're using db search you'd need to search using a wild card and then paginate through the users looking for `active:false` on the user object.

**Comment by soullivaneuh** [comment]
Yes the `active:false` seems to solve my issue:

![image](https://user-images.githubusercontent.com/1698357/92325295-b0efb600-f049-11ea-8770-3fc1d4a96fb1.png)

But having a built-in UI filter would be a nice addition in my opinion.

And in general, it's not obvious that you can use elastic-search filter directly here.

Some UX improvement with a smart filter bar like its done on GitHub or GitLab would be also nice. :+1: 

> Are you looking for a new search control for "Active" or "Locked" users?

I supposed locked user but in that case the filter name looks wrong to me. What is the difference between "active" and "locked" to you?

**Comment by robotdan** [comment]
> Some UX improvement with a smart filter bar like its done on GitHub or GitLab would be also nice. 👍

Yes! This has been on our list for a while. This is something we plan to do. (related issue https://github.com/FusionAuth/fusionauth-issues/issues/30#issuecomment-437723789)

Internal notes:

Until we have built out the intelli-sense search field, we could think about adding a checkbox or other search control to the advanced section to find locked users. `active:false` since this isn't super obvious.
